test/functional/feature_fee_estimation.py

```diff
@@ -168,6 +168,11 @@ def transact_and_mine(self, numblocks, mining_node):
                     newmem.append(utx)
             self.memutxo = newmem
 
+    def import_deterministic_coinbase_privkeys(self):
+        self.start_nodes()
```

Comment:
Couldn't the test framework implementation of `import_deterministic_coinbase_privkeys` be smart enough to know whether nodes are running or not with `node.is_node_stopped` (or other state)? It seems unfortunate that an individual test would need to override the `import_deterministic_coinbase_privkeys` method to get such basic functionality.